index.html

```diff
@@ -216,25 +119,75 @@ <h3 class="project-title">
               alt=""
             />
           </div>
-          <div class="model-body">
-            <h2 class="model-text">Keeping track of hundreds of components</h2>
-            <div class="mobile-list">
-              <ul>
-                <li>Ruby on Rails</li>
-                <li>CSS</li>
-                <li>JavaScript</li>
-              </ul>
-            </div>
-            <div class="desktop-list">
-              <ul>
-                <li>Codekit</li>
-                <li>GitHub</li>
-                <li>JavaScript</li>
-                <li>Bootstrap</li>
-                <li>Terminal</li>
-                <li>Codepen</li>
-              </ul>
+          <div class="model-container">
+            <div class="model-body">
+              <h2 class="model-text">
+                Keeping track of hundreds of components
+              </h2>
+              <div class="mobile-list">
+                <ul>
+                  <li>Ruby on Rails</li>
+                  <li>CSS</li>
+                  <li>JavaScript</li>
+                </ul>
+              </div>
+
+              <div class="desktop-list">
+                <ul>
+                  <li>Codekit</li>
+                  <li>GitHub</li>
+                  <li>JavaScript</li>
+                  <li>Bootstrap</li>
+                  <li>Terminal</li>
+                  <li>Codepen</li>
+                </ul>
+              </div>
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it 1960s.
+              </p>
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it 1960s.
+              </p>
             </div>
+          </div>
+          <div class="model-footer desktop-flex">
+            <button class="btn-live">
+              See live
+              <span
+                ><i
+                  class="fa-thin fa-arrow-up-left-from-circle fa-rotate-90"
+                ></i
+              ></span>
+            </button>
+            <button class="btn-source">
+              See source <span><i class="fa-brands fa-github"></i></span>
+            </button>
+          </div>
+        </div>
+      </div>
+      <div id="overlay"></div>
+    </section>
```

Comment:
- [ ] Hi @Umuhan, based on the requirements of this project, this information on your pop-up should have been created dynamically from the projects array in your JS file not from your HTML file as it is the case now. All the HTML of this section should be created when the page loads. This section of the HTML codes should be removed and added dynamically via JavaScript. The reason for adding projects from the projects array is to make it very easy to add more projects in the future. Kindly adhere to the requirements.
- [ ] Again you'll make sure your design is exactly the same as in Figma. There are some differences on the position of technologies, see live, and see sources buttons. Kindly check the differences in the 
screenshots and fix them

| Figma :+1:  | Yours |
| --- | --- |
| ![Screenshot from 2023-05-23 13-47-50](https://github.com/Umuhan/My-Portfolio-Project/assets/63087491/1c7e6da1-6034-409f-a9a7-f31756a83e6d) | ![Screenshot from 2023-05-23 13-49-24](https://github.com/Umuhan/My-Portfolio-Project/assets/63087491/9f7b17f9-80d3-4731-9604-a1dd1fade65d) |

- For a good user experience, let's remove the scrolling effect. The pop-up should fill the screen
- Also, the mobile version should be exactly the same as in Figma.

Reply:
done.